docs/_sidebar.md

```diff
@@ -39,6 +37,8 @@
 
 - Advanced Topics
   - [Personal Access Tokens](20_personal_access_token.md)
+  - [GitHub Actions](17_github_actions.md)
+  - [Automate Releases](17_actions_activity.md)
 
 - Appendix
   - **GitHub Resources**
```

Comment:
```suggestion
- Appendix
  - **Advanced Topics**
    - [Personal Access Tokens](20_personal_access_token.md)
    - [GitHub Actions](17_github_actions.md)
    - [Automate Releases](17_actions_activity.md)
  - **GitHub Resources**
```

I suggest moving Advanced Topics to the appendix, but I’m open to other thoughts and opinions. 🙂

Reply:
I was thinking that I need this in Advanced because of Experts. @beardofedu thoughts?